scripts/load_env.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Why Bash and not JavaScript? Not everyone on the team is proficient with Bash, which makes this script less maintainable than it could be.
I understand you had to use AWS CLI, but you have use Node's `child_process`.